src/components/main/actionsPanel/nodeTreeView/hooks/useNodeViewState.ts

```diff
@@ -20,21 +21,20 @@ export function useNodeViewState() {
     prevRenderableFileUid,
 
     validNodeTree,
-    nSelectedItems: selectedItems,
     nSelectedItemsObj: selectedItemsObj,
   } = useAppState();
-  const { addRunningActions, removeRunningActions } = useContext(MainContext);
 
   const cb_focusNode = useCallback(() => {
-    removeRunningActions(["nodeTreeView-focus"]);
-  }, [removeRunningActions]);
+    dispatch(addRunningAction());
+    dispatch(removeRunningAction());
```

Comment:
@VictoriaShyika this is not making any sense!
Are we not doing anything inside it?